docs/UserGuide.md

```diff
@@ -342,6 +349,7 @@ Adds an event to nustracker
 Command: `create n/EVENT_NAME d/EVENT_DATE t/EVENT_TIME`
 
 - Event name cannot be empty.
+- Event name cannot contain the backslash character as backslashes are used to identify prefixes.
```

Comment:
I'm not really sure about this, but do you mean forward slashes?

Reply:
Ok ya fixed